src/utils.js

```diff
@@ -6,12 +6,17 @@ import fsExtra from 'fs-extra';
 import ApifyClient from 'apify-client';
 import psTree from 'ps-tree';
 import pidusage from 'pidusage';
+import requestPromise from 'request-promise';
 import _ from 'underscore';
+import XRegExp from 'xregexp';
 import { delayPromise } from 'apify-shared/utilities';
 import { ENV_VARS } from './constants';
 
 export const PID_USAGE_NOT_FOUND_ERROR = 'No maching pid found';
 
+export const URL_NO_COMMAS_REGEX = XRegExp('https?://(www\\.)?[-\\p{L}0-9@:%._\\+~#=]{2,256}\\.[a-z]{2,6}\\b([-\\p{L}0-9@:%_\\+.~#?&//=]*)', 'gi');
```

Comment:
Where are these regexps from? It doesn't seem to me that they validate the DNS hostname correctly - they would consider https://-aaa-/xxx as valid URL.

Reply:
StackOverflow or some other forum. I needed to try the Unicode flags `p{L}` so I figured I'll just grab one that successfully matches the list of URLs crawled by the user who reported the bug on Intercom and then go from there.